plugins/backend/backstage-plugin-argo-cd-backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@roadiehq/backstage-plugin-argo-cd-backend",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
```

Comment:
When we publish this NPM will handle linking etc. stuff based on the repository information. Could add a block for those in here. Something like:

  "repository": {
    "type": "git",
    "url": "github:RoadieHQ/roadie-backstage-plugins",
    "directory": "plugins/backend/backstage-plugin-argo-cd-backend"
  },

Reply:
addressed